pkg/helm/v3/converter.go

```diff
@@ -0,0 +1,50 @@
+package v3
+
+import (
+	"github.com/helm/helm-2to3/pkg/common"
+	helm2to3v2 "github.com/helm/helm-2to3/pkg/v2"
+	helm2to3v3 "github.com/helm/helm-2to3/pkg/v3"
+)
+
+// Converter Converts a given helm 2 release with all its release versions to helm 3 format and deletes the old release from tiller
+type Converter struct {
+	TillerNamespace string
+}
+
+// Convert attempts to convert the given release name from v2 to v3.
+func (c Converter) Convert(releaseName string) error {
+	retrieveOpts := helm2to3v2.RetrieveOptions{
+		ReleaseName:      releaseName,
+		StorageType:      "configmaps",
+		TillerLabel:      "",
+		TillerNamespace:  c.TillerNamespace,
+		TillerOutCluster: false,
```

Comment:
```suggestion
		ReleaseName:      releaseName,
		TillerNamespace:  c.TillerNamespace,
```

(others are defaults)